components/server/src/analytics.ts

```diff
@@ -16,7 +16,7 @@ export async function trackLogin(
     subscriptionService: SubscriptionService,
 ) {
     // make new complete identify call for each login
-    fullIdentify(user, request, analytics, subscriptionService);
+    await fullIdentify(user, request, analytics, subscriptionService);
```

Comment:
I don't think we should make everyone wait for this -- i.e. this call is probably fine to leave async.

We could silence the warning though (don't remember if that's with `/** no await */` or something else)

Reply:
Ah, indeed, thanks for pointing this out. 👍